core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixTextToSpeech.kt

```diff
@@ -135,9 +133,7 @@ class KiwixTextToSpeech internal constructor(
     languageAvailabilityResult == LANG_MISSING_DATA ||
       languageAvailabilityResult == LANG_NOT_SUPPORTED
 
-  private fun getFeatures(
-    tts: TextToSpeech?
-  ): Set<String> = tts!!.voice.features
+  private fun getFeatures(tts: TextToSpeech?): Set<String> = tts!!.voice.features
```

Comment:
ah also null safe this, return an empty set using an elvis operator if we don't get anything

Reply:
but also tts should not be null in the first place so it shouldn't matter?